From b72e24bd6eb47e8f00cd2caea85d52625391a75f Mon Sep 17 00:00:00 2001
From: Noam Lessner <nlessner@habana.ai>
Date: Tue, 28 Jan 2020 22:42:26 +0200
Subject: [PATCH] [SW-8228] goya2: pldm: enable pci link up

return 0 so we will not skip bringing up the pci
link on paladium platform

Change-Id: I09f2ef2914169fb8d03dcf3be21b6dc1cba4844c
Signed-off-by: Noam Lessner <nlessner@habana.ai>
---

diff --git a/goya2_arc/pldm.c b/goya2_arc/pldm.c
index b2a204d..d33c141 100644
--- a/goya2_arc/pldm.c
+++ b/goya2_arc/pldm.c
@@ -27,7 +27,7 @@
 
 u32 hl_skip_pcie_init(void)
 {
-	return 1;
+	return 0;
 }
 
 u64 hl_pll_get_cpu_cycles_per_usec(void)
